### DISCUSSION TITLE
Exploring plotnine features with geospatial data

### DISCUSSION THREAD

**Discussion by sporella** [has2k1/plotnine#860] [2024 Plotnine Contest]
### Authors

Stephanie Orellana Bello

### Links

https://github.com/sporella

### Full description

For this Plotnine Contest I wanted to explore the features of plotnine for geospatial data visualization. I also wanted to prove if I can replicate my usual ggplot2 workflow and if my common used features are available in plotnine.

Here the results:

![](https://sporella.github.io/p9c/index_files/figure-html/cell-4-output-1.png)
![](https://sporella.github.io/p9c/index_files/figure-html/cell-5-output-1.png)

For reproducible code, explanation and comments see: https://sporella.github.io/p9c

### Code repository

https://github.com/sporella/p9c

**Comment by has2k1** [comment]
![map](https://github.com/user-attachments/assets/01662405-d917-465f-ad28-e9668781db51)
<hr/>

![column](https://github.com/user-attachments/assets/7a3ffaa5-3b9d-42bc-bbda-46bf84b85e67)

**Comment by EconomiCurtis** [comment]
Ahoy @sporella , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by sporella** [comment]
Thanks @EconomiCurtis !!!

**Comment by has2k1** [comment]
@sporella, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
